src/F500/CI/Task/Codeception/CodeceptionTask.php

```diff
@@ -72,6 +72,10 @@ protected function createRunCommand(CommandFactory $commandFactory)
         if (!empty($options['suite'])) {
             $specificSuite = $options['suite'];
         } elseif (!empty($options['skip_suites'])) {
+            if (!is_array($options['skip_suites'])) {
```

Comment:
No real reason; mainly because this is more explicit and relies less on language magic :)